versions.yaml

```diff
@@ -279,6 +279,16 @@ languages:
         building Kata
       newest-version: "1.58.1"
 
+  golangci-lint:
```

Comment:
Does this change have anything to do with fixes?

Reply:
No. But it is needed to pass the CI